### ISSUE TITLE
Couple of Recommendations.... and discovered something EPIC...!!!

### ISSUE THREAD

**Issue by leeuniverse** [moktavizen/material-darker-jdownloader#11]
**First,** as you can see with your Text/Spacing etc. UI, in the middle etc. compared to the default or Black Eye Theme does it, yours is WAY TOO BIG.
I would highly recommend you make the Text and Spacing etc. of yours much "Smaller".
BTW, it doesn't have to be "as" small as the Default or my example.  I DO think my example that stuff could be "slightly" larger, but yours is FAR too "in the face".

**Second,** I would recommend you use Vinylwalk3r's "Green Progress Bars"...  They would look much better than your white ones.
And they are a good example, look how MASSIVE your progress bars are....  WAY too much large text and spacing etc.
https://github.com/Vinylwalk3r/JDownloader-2-Dark-Theme
You can download his progress bars separately on his website, and I think you can see in his JAR file "where" they actually go to replace the default ones.

**Third,** it looks like you didn't correctly add your "images"?  You're replacing Jdownloaders when we do a manual install.
I think you need to put the image folder in the JAR file....  For example, if you look at Vinylwalk3r's "syntheticaBlackEye.jar" that you download from his site, you can see he has replacement images in there.  Though, I don't know if that works for all your Icons etc.?  Just checking with you.

**Fourth, you replacing Jdownloaders default Icons created a "Happy Accident"....**
So, because I didn't like how "big" your text and spacing etc. was, I decided to switch back for the moment to Vinylwalk3r's edited Black Eye Theme.  Here's what I did....

> 1. Had Black Eye Theme selected.
> 
> 2. "Window Decoration" selected in Advanced Settings.
> 
> 3. Replaced "syntheticaBlackEye.jar" with the one from here: https://github.com/Vinylwalk3r/JDownloader-2-Dark-Theme
> 
> 4. Manual Installed the "Material Darker JDownloader" Theme, changing to it in Advanced Settings.  Here: https://github.com/moktavizen/material-darker-jdownloader
> 5. Didn't really like it, so simply switched in Settings back to "Black Eye" and HOLY ****...  Look at these picks.
> 
> So, I think he did the Theme wrong and he "overwrites" the Default Jdownloader Images, instead of putting the image folder in the "flatlaf.jar" file.  I don't know, don't edit themes, just saw that.
> 
> Anyway,, look how massively NICE Jdownloader looks using the the Black Eye edited version and Material Darkers Icons and I'm guessing that's it?
> It looks absolutely AMAZING doesn't it...?
> 
> If you're willing to take the time, I think you should seriously make this the "Default" Theme of Jdownloader, if the parties in question are okay with it.  I mean WOW.

If you look at the attached images I think I just created **### the BEST Theme design EVER for Jdownloader.**

What do you think....?
1st image yours.
2nd & 3rd Image your icons with modified Black Eye Theme.
(also, some of the above text is a response to Jdownloader Support, was talking to them about something, so just shared with them what I discovered.)

BTW, I WOULD like the text spacing, progress bars, and the text "color" to be slightly "larger" than my images show.
It would be nice if those things were slightly more readable.  But yours IS far too big.

![1](https://github.com/user-attachments/assets/f4522640-71ac-4161-b03e-6ef265c49330)
![2](https://github.com/user-attachments/assets/9be21d64-9b3d-4780-84d7-fe8d1d757d49)
![3](https://github.com/user-attachments/assets/d6243e9a-2fb6-4912-b603-d68ac3fb17d4)

**Comment by leeuniverse** [comment]
**UPDATE....**

Okay, so I know what was "still" Irritating me about your Icon changes, even though I LOVED the new "combo" I accidentally created, it STILL wasn't "as good" as the Default Black Eye Theme.

I needed to "change" certain Icons BACK to their Jdownloader Defaults.
See Screenshot...

- The "Extraction Ok" icon.  Back to default.
- The "Move Download Up/Down the List Icons.  Back to Default.
- And the "Update" icon.  Back to Default.
(ignore the arrow on the line, that's an issue discussing with Jdownloader)

So, in conclusion....  I think that THIS is now the BEST Theme EVER created for Jdownloader, a combination of your Icons with the edited "Black Eye" by the other creator, and the changes above.
I LOVE THIS....

BTW, I'm looking for those Icons to get them "updated" with larger sized like yours.
Found the Arrrows and likely the RAR, now looking for the "Update" icon.

**Comment by 4NXIE7Y** [comment]
> **UPDATE....**
> 
> Okay, so I know what was "still" Irritating me about your Icon changes, even though I LOVED the new "combo" I accidentally created, it STILL wasn't "as good" as the Default Black Eye Theme.
> 
> I needed to "change" certain Icons BACK to their Jdownloader Defaults. See Screenshot...
> 
>     * The "Extraction Ok" icon.  Back to default.
> 
>     * The "Move Download Up/Down the List Icons.  Back to Default.
> 
>     * And the "Update" icon.  Back to Default.
>       (ignore the arrow on the line, that's an issue discussing with Jdownloader)
> 
> 
> So, in conclusion.... I think that THIS is now the BEST Theme EVER created for Jdownloader, a combination of your Icons with the edited "Black Eye" by the other creator, and the changes above. I LOVE THIS....
> 
> BTW, I'm looking for those Icons to get them "updated" with larger sized like yours. Found the Arrrows and likely the RAR, now looking for the "Update" icon.
> 
> FOUND.... https://www.iconfinder.com/icons/118827/update_system_software_icon https://pixabay.com/vectors/arrow-bottom-go-icon-icons-matt-1294467/ https://pixabay.com/vectors/arrow-down-go-icon-icons-matt-1294465/
> 
> And places we could maybe find a RAR Archive image to use. https://www.pngwing.com/en/search?q=winrar https://www.pngegg.com/en/search?q=winrar https://www.google.com/search?sa=N&sca_esv=9a213b508c211de4&sca_upv=1&udm=2&biw=2187&bih=1083&cs=1&q=winrar+logo+png&stick=H4sIAAAAAAAAAFvEyl-emVeUWKSQk5-er1CQlw4AV6BBoRIAAAA&source=univ&ved=2ahUKEwisqe2lreyIAxXLle4BHRZQC2c4FBCs3AJ6BAhjEAA
> 
> Now we just need to "edit" them.... colors, size, other images part of the icons, make the additionals etc.
> 
> ![edit](https://private-user-images.githubusercontent.com/9868451/372322885-da932570-beb4-4011-a8d6-dd29e1506743.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ2MDg4MTcsIm5iZiI6MTczNDYwODUxNywicGF0aCI6Ii85ODY4NDUxLzM3MjMyMjg4NS1kYTkzMjU3MC1iZWI0LTQwMTEtYThkNi1kZDI5ZTE1MDY3NDMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTlUMTE0MTU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTczNjUyYmMyN2JkZDM3ZDMwYTI4NmZmMzMyOWM3ZGE3ZWMzNTI0YzU3ODkxYzUwNmRiMDY0MmQ5NTc3NmFkNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.fpcXOOJ8Zkt7g4RT_N9JFRpXMTUR1uvCHwOXCvda1GA)

Is this going to be available publicly anytime soon?

**Comment by moktavizen** [comment]
Glad that you found a way to make JDownloader looks like how you wanted it to be. The great thing about JDownlaoder is that you can almost customize anything if you put in the effort. 

Regarding the progress bar color, it depends on the color of the table foreground, I can't change the progress bar image because the base is FlatLaf, not Black Eye Theme, and if I change the base to Black Eye a lot of components will look 3D and not Flat styled, which don't match with the intended purpose of this theme.
